backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/domain/UserPatch.java

```diff
@@ -0,0 +1,13 @@
+package ca.gov.dtsstn.cdcp.api.service.domain;
+
+import org.immutables.value.Value.Immutable;
+
+import jakarta.annotation.Nullable;
+
+@Immutable
+public interface UserPatch extends BaseDomainObject {
```

Comment:
I don't think this object should subclass `BaseDomainObject`.

When you are updating a stored object, the fields within `BaseDomainObject` should be readonly. It doesn't make sense to allow for them to be set. Just remove the `extends BaseDomainObject` and you should be okay.

Reply:
this class is removed